sidecar/src/rest/onNewProject.ts

```diff
@@ -0,0 +1,43 @@
+import { Router } from "express";
+import { Team } from '../entity/Team.entity';
+import { TeamMember } from '../entity/TeamMember.entity';
+import { Employee } from "../entity/Employee.entity";
+
+const api = Router();
+
+api.post("/on-new-project", async (req, res) => {
+  const {
+    event: { data: { new: project } },
+  } = req.body;
+
+
+  const employeeRepo = Employee.getRepository();
+
+  // if admin creates a project than there is not need to create a team automatically since admin has no ecode
+  const user = await employeeRepo.findOneOrFail({ user_id: project.created_by });
+  if (!user.ecode) {
+    res.status(201).send();
```

Comment:
We should add an explicit return statement here. There is no guarantee that `res.send` will stop the execution right here.